### PR TITLE
TASK: Adjust neos/neos css build to work on apple silicon

### PR DESCRIPTION
**Pull request by kitsunet**

### CODE REVIEW

**Review by jonnitto — APPROVED** [review]
Can't wait till we ged rid of sass 🤣

File: Neos.Neos/package.json [hunk 1]
```diff
@@ -18,10 +18,11 @@
     "babel-loader": "^8.2.2",
     "compass-mixins": "^0.12.10",
     "css-loader": "^5.1.1",
+    "css-minimizer-webpack-plugin": "~4.0",
```
Comment (mhsdesign) [inline Neos.Neos/package.json]:
```suggestion
```

... and also yarn lock

**Review by mhsdesign — APPROVED** [review]
Thanks for taking care ❤️ 

How should we upmerge this? Rebuild and commit on every branch?

**Comment by kitsunet** [comment]
> Thanks for taking care ❤️
> 
> How should we upmerge this? Rebuild and commit on every branch?

Yeah probably necessary. I would just merge this later today and directly do the upmerges as well.

**Comment by kitsunet** [comment]
all upmerged

**Pull request merged**